pipelines/fbc-release/README.md

```diff
@@ -19,6 +19,9 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                    | Yes       | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | No        | -                                                               |
 
+## Changes in 4.1.0
+* Implement the `check-data-keys` task to validate the `data.json` file using the JSON schema.
```

Comment:
I would suggest to use the verb "Add" instead of "Implement"  - because that's what being done, right? We're adding the new task. (This applies to all the others as well.)

Reply:
Yeah, we are adding .. will change it